OpenWord-MMO/Program.cs

```diff
@@ -0,0 +1,30 @@
+﻿
+using System;
+using System.Net;
+using System.Net.Sockets;
+using System.Text;
+using System.Text.RegularExpressions;
+
+var serverEndpoint = new IPEndPoint(IPAddress.Loopback, 14411);
+
+var server = new UdpClient(serverEndpoint);
+Regex regex = new Regex(@"^\S{0,20}$");
+string text = "";
+
+while (true){
+    IPEndPoint clientEndPoint = default;
+    var clientData = server.Receive(ref clientEndPoint);
+    var clientText = Encoding.ASCII.GetString(clientData);
+    var errorMessage = Encoding.ASCII.GetBytes("ERROR! Message is either over 20 characters or contains wide space, try again");
+    
+    Console.WriteLine($"packet received from: {clientEndPoint} saying: {clientText}");
+    
+    if (!regex.IsMatch(clientText))
+        server.Send(errorMessage, clientEndPoint);
+    
+    if (regex.IsMatch(clientText)){
```

Comment:
Use `else` or `continue`, but don't evaluate the RegEx a second time, for performance reasons :)